cmd/server/assets/realmadmin/_form_general.html

```diff
@@ -55,6 +56,41 @@
     </small>
   </div>
 
+  <hr>
+  <div class="form-group">
+    <div class="form-check">
+      <input type="checkbox" name="allow_key_server_stats" id="allow-key-server-stats" class="form-check-input"
+        value="true" {{checkedIf (ne $statsConfig.RealmID 0)}} />
+      <label for="allow-bulk-true" class="form-check-label">
+        Enable key-server statistics
+        <small class="form-text text-muted mb-3">
+          Checking this box will allow this realm to collect statistics from the key-server about codes
```

Comment:
there needs to be a note here about how this may need to be disclosed your applications privacy policy.

we should also describe what's here -onset to upload, uploads by operating system.